{
  "Header": {
    "Original": "CVE-2018-3721 (lodash node module before 4.17.5 suffers from a Modification of Assume ...)",
    "Line": 442366,
    "ID": "CVE-2018-3721",
    "Description": "(lodash node module before 4.17.5 suffers from a Modification of Assume ...)"
  },
  "Annotations": [
    {
      "Original": "- node-lodash 4.17.11+dfsg-1 (unimportant; bug #890575)",
      "Line": 442367,
      "Type": "package",
      "Package": "node-lodash",
      "Kind": "fixed",
      "Version": "4.17.11+dfsg-1",
      "Severity": "unimportant",
      "BugNo": 890575
    },
    {
      "Original": "NOTE: https://snyk.io/vuln/npm:lodash:20180130",
      "Line": 442368,
      "Type": "NOTE",
      "Description": "https://snyk.io/vuln/npm:lodash:20180130"
    },
    {
      "Original": "NOTE: https://github.com/lodash/lodash/commit/d8e069cc3410082e44eb18fcf8e7f3d08ebe1d4a",
      "Line": 442369,
      "Type": "NOTE",
      "Description": "https://github.com/lodash/lodash/commit/d8e069cc3410082e44eb18fcf8e7f3d08ebe1d4a"
    },
    {
      "Original": "NOTE: nodejs not covered by security support",
      "Line": 442370,
      "Type": "NOTE",
      "Description": "nodejs not covered by security support"
    }
  ]
}